build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -13,6 +13,20 @@
                            Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
         <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
       </NupkgsForPublishing>
+
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.swr"
```

Comment:
Wait... this isn't a nupkg so I'm removing it, but I don't quite remember what it is.

Reply:
This file is needed along with the nupkg if the files change inside the nupkg. That is unlikely to ever happen in 2.x again, though. It is a manual process. As long as the swr is available somewhere, it's ok.

Reply:
Ok, cool. Yeah, it'll still be available in the blob feed as an asset, which makes it end up on the internal file share.